src/Bicep.LangServer.IntegrationTests/HoverTests.cs

```diff
@@ -339,7 +339,7 @@ public async Task Hovers_are_displayed_on_description_decorator_objects()
                 h => h!.Contents.MarkupContent!.Value.Should().EndWith("```\nthis is my module\n"),
                 h => h!.Contents.MarkupContent!.Value.Should().EndWith("```\nthis is my param\n"),
                 h => h!.Contents.MarkupContent!.Value.Should().EndWith("```\nthis is my var\n"),
-                h => h!.Contents.MarkupContent!.Value.Should().EndWith("```\nthis is my  \nmultiline  \nresource  \n[View Documentation](https://docs.microsoft.com/azure/templates/test.rp/discriminatortests?tabs=bicep)\n"),
+                h => h!.Contents.MarkupContent!.Value.Should().EndWith("```\nthis is my  \nmultiline  \nresource  \n[View Documentation](https://learn.microsoft.com/azure/templates/test.rp/discriminatortests?tabs=bicep&pivots=deployment-language-bicep)\n"),
```

Comment:
```suggestion
                h => h!.Contents.MarkupContent!.Value.Should().EndWith("```\nthis is my  \nmultiline  \nresource  \n[View Documentation](https://learn.microsoft.com/azure/templates/test.rp/discriminatortests?pivots=deployment-language-bicep)\n"),
```

It looks like the `pivots` querystring parameter replaced the `tabs` one altogether (rather than supplementing it):

![image](https://github.com/Azure/bicep/assets/705500/d344f971-c10f-4c78-bccc-03fe23c63752)

Reply:
Ah, removed the `tabs` parameter now.